plugin.yaml

```diff
@@ -9,3 +9,9 @@ navigation:
         label: 'iPrice API'
         url: /ipricegroup/ocapiplugin/resources
         icon: icon-cloud
+        permissions:
+        - iprice-restful-api-generator
```

Comment:
Please indent properly. Also, I think it's better to give it a more generic permission: `manage_apis`